cms/djangoapps/contentstore/views/tests/test_import_export.py

```diff
@@ -556,3 +629,41 @@ def test_export_success_with_custom_tag(self):
         )
 
         self.test_export_targz_urlparam()
+
+
+@override_settings(CONTENTSTORE=TEST_DATA_CONTENTSTORE)
+class TestLibraryImportExport(CourseTestCase):
+    """
+    Tests for importing content libraries from XML and exporting them to XML.
+    """
+    def setUp(self):
+        super(TestLibraryImportExport, self).setUp()
+        self.export_dir = tempfile.mkdtemp()
+        self.addCleanup(shutil.rmtree, self.export_dir, ignore_errors=True)
+
+    def test_content_library_export_import(self):
+        library = LibraryFactory.create(modulestore=self.store)
+        source_library_key = library.location.library_key
+
+        import_library_from_xml(
+            self.store,
+            'test_user',
+            TEST_DATA_DIR,
+            ['library_empty_problem'],
+            static_content_store=contentstore(),
+            target_id=source_library_key,
+            load_error_modules=False,
+            raise_on_failure=True,
+            create_if_not_present=True,
+        )
+
+        export_library_to_xml(
+            self.store,
+            contentstore(),
+            source_library_key,
+            self.export_dir,
+            'exported_source_library',
+        )
+
+        source_library = self.store.get_library(source_library_key)
+        self.assertEqual(source_library.url_name, 'library')
```

Comment:
can we check that a library is unchanged between a first and second import? We have a helper to check if courses are equal: [`assertCoursesEqual`](https://github.com/edx/edx-platform/blob/master/cms/djangoapps/contentstore/tests/utils.py#L278). I'm guessing we can alter that for libraries.

I ask this because in a previous attempt to fix this bug, we ran into an issue where export/import would change a library's content.